tests/test-webapps/test-jndi-webapp/src/main/config/modules/demo.d/demo-jndi.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0"?><!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "https://www.eclipse.org/jetty/configure_9_3.dtd">
```

Comment:
Must be `configure_10_0.dtd`.

Reply:
Fixed in 1feb60bb2c8e521f80aaee2abc901c067b561026 (for many files in `/demos/`)